source/common/http/status.cc

```diff
@@ -21,6 +21,8 @@ absl::string_view statusCodeToString(StatusCode code) {
     return "PrematureResponseError";
   case StatusCode::CodecClientError:
     return "CodecClientError";
+  default:
+    RELEASE_ASSERT(false, "Invalid StatusCode value");
```

Comment:
Please use NOT_REACHED_GCOVR_EXCL_LINE macro. It will exclude this code path from test coverage.

Reply:
it's not coverage that is the issue, just compilation

Reply:
if that macro solves compilation we can swap it out

Reply:
The macro will cause an abort(). Does that suffice?

Reply:
yeah, hadn't looked into the macro definition but it just induces a panic so that works great